.github/workflows/test.yml

```diff
@@ -0,0 +1,59 @@
+name: Build, Test and Optionally Deploy
+
+on: [push, pull_request, workflow_dispatch]
+
+# According to: https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md
+# Both Node and Yarn should be installed already
+# This would ideally be split up into dependent jobs maybe, but this is a direct conversion of the travis script for now
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
```

Comment:
maybe lets set this to the same version as server

Reply:
Good call, I wasn't sure if there's a way to do this! But after doing a search for other people's code it looks like there is :)